inbm/dispatcher-agent/tests/unit/source/test_ubuntu_source_cmd.py

```diff
@@ -218,11 +218,11 @@ def test_add_app_deb_822_format_successfully(self):
             params = ApplicationAddSourceParameters(
                 source_list_file_name="google-chrome.sources",
                 sources=["X-Repolib-Name: Google Chrome",
-                        "Enabled: yes",
-                        "Types: deb",
-                        "URIs: https://dl-ssl.google.com/linux/linux_signing_key.pub",
-                        "Suites: stable",
-                        "Components: main"],
+                         "Enabled: yes",
+                         "Types: deb",
+                         "URIs: https://dl-ssl.google.com/linux/linux_signing_key.pub",
+                         "Suites: stable",
+                         "Components: main"],
```

Comment:
<h3>$\color{#FFD700}Coding Style: CodeConsistency$</h3>

Verify that the formatting changes in the test cases do not affect the execution and outcome of the tests.

Reply:
there is no way this could affect the outcome of the tests as it is python syntax level formatting